addon/components/polaris-text-style.js

```diff
@@ -60,33 +59,33 @@ export default Component.extend({
    * @default: null
    * @public
    */
-  text: null,
+  text = null;
 
   /**
    * Custom style classes to apply to this element.
    *
    * @type {String}
    */
-  classes: '',
-
-  dataTestTextStyle: true,
+  classes = '';
```

Comment:
can you add a deprecation for this please like we did for [others](https://github.com/smile-io/ember-polaris/pull/360/files#diff-376df1f15a1defdda443249bea7c9951R244) in these PRs